### PR TITLE
Polish readme with architecture and feature list sections.

### PR DESCRIPTION
**Pull request by weihanglo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,100 @@ disk manager help to manage host disks, implementing disk partition and file sys
...
+- [x] Disk discovery, including existing block devices, and hot plugged disks.
+- [x] Support multiple storage controller (IDE/SATA/SCSI/Virtio).
+- [x] Support vritual disks (WWN on the disk is required for unique identification).
+- [ ] Device mapper and LVM are not yet supported.
+- [ ] The behaviour of multipath devices is undefined.
+
+## Architecture
+
+The **Node Disk Manager** (a.k.a. NDM) is a simple kubernetes controller,
+following the famous [controller pattern]. It leverages Rahcher's [wrangler]
```
Comment (bk201) [inline README.md:27]:
```suggestion
following the famous [controller pattern]. It leverages Rancher's [wrangler]
```
```suggestion
following the famous [controller pattern]. It leverages Rahcher's [wrangler]
```

Comment (bk201) [inline README.md:27]:
```suggestion
following the famous [controller pattern]. It leverages Rancher's [wrangler]
```
```suggestion
following the famous [controller pattern]. It leverages Rahcher's [wrangler]
```

**Review by bk201 — APPROVED** [review]
LGTM!

**Review by Vicente-Cheng — APPROVED** [review]
Thanks for the detailed document :)

**Review by guangbochen — APPROVED** [review]

**Pull request merged**